### PR TITLE
doc: mizar network performance test

### PR DESCRIPTION
**Pull request by h-w-chen**
This PR is the result of finishing #347 
This is How-to document for mizar network performance metrics collecting process. 
It also includes one set of sample metrics got from AWS Kind 1+2 env.

### CODE REVIEW

File: docs/dev/network-perf-test.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# How to test Mizar network performance
+This document records the considerations of Mizar network performance testing, and the tools used and steps of processes in metric collecting.
+
+Also, it has one sample collection of network metrics for the reference, which was the result of one of the basic performace testings.
```
Comment (clu2xlu) [inline docs/dev/network-perf-test.md]: performance

File: docs/dev/network-perf-test.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+
+## Background
+Mizar is based on new technology of eBPF/XDP, provides brand new networking for containers. In order to evaluate Mizar's advantages, systematic process is desired to collect its network performance metric data, and compare with other network providers like Cilium and Calico.  
+
+
+## Target network metrics
+* TCP stream throuput (not to confuse with bandwidth)
+* UDP PPS throughput
+* Latency
```
Comment (clu2xlu) [inline docs/dev/network-perf-test.md]: Network latency?

File: docs/dev/network-perf-test.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+Also, it has one sample collection of network metrics for the reference, which was the result of one of the basic performace testings.
+
+
+## Background
+Mizar is based on new technology of eBPF/XDP, provides brand new networking for containers. In order to evaluate Mizar's advantages, systematic process is desired to collect its network performance metric data, and compare with other network providers like Cilium and Calico.  
+
+
+## Target network metrics
+* TCP stream throuput (not to confuse with bandwidth)
```
Comment (clu2xlu) [inline docs/dev/network-perf-test.md]: throughput

File: docs/dev/network-perf-test.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+```bash
+iperf3 -c iperf-s -t 60
+```
+To get UDP PPS metrics:
+```bash
+iperf3 -c iperf-s -t 60 -u -b 1000M -l 64
+```
+After the throughput test is done, delete iperf3 server and client pods.
+
+### Latency testing
```
Comment (clu2xlu) [inline docs/dev/network-perf-test.md]: what kind of latency? network latency?

**Review by clu2xlu — CHANGES_REQUESTED** [review]

File: docs/dev/network-perf-test.md (hunk 1 above)
Comment (h-w-chen) [reply docs/dev/network-perf-test.md]: thanks for pointing typo out. will fix.

File: docs/dev/network-perf-test.md (hunk 2 above)
Comment (h-w-chen) [reply docs/dev/network-perf-test.md]: will add network

File: docs/dev/network-perf-test.md (hunk 4 above)
Comment (h-w-chen) [reply docs/dev/network-perf-test.md]: willl add a bit more information

File: docs/dev/network-perf-test.md (hunk 3 above)
Comment (h-w-chen) [reply docs/dev/network-perf-test.md]: willl fix the typo

**Review by clu2xlu — APPROVED** [review]

File: docs/dev/network-perf-test.md [hunk 5]
```diff
@@ -39,8 +39,8 @@ iperf3 -c iperf-s -t 60 -u -b 1000M -l 64
 ```
 After the throughput test is done, delete iperf3 server and client pods.
 
-### Latency testing
-netperf tool is used to collect latency metrics.
+### Metwork latency testing
```
Comment (clu2xlu) [inline docs/dev/network-perf-test.md]: Network

**Review by clu2xlu — APPROVED** [review]

**Pull request merged**